sdk/storage/Azure.Storage.Queues/src/QueueClient.cs

```diff
@@ -1835,6 +2008,20 @@ private void SetNameFieldsIfNull()
                 }
             }
         }
+
+        private QueueMessage ToQueueMessage(DequeuedMessageItem dequeuedMessageItem)
```

Comment:
Nit - consider moving this onto your QueueMessage class so it's easier to remember to update the properties whenever somebody adds a new one?